conda-recipes/iqsharp/meta.yaml

```diff
@@ -24,9 +24,9 @@ requirements:
 
 test:
   requires:
-    - jupyter
+    - notebook=6.0.1
     - python>=3.6
-    - jupyter_client
+    - jupyter_client<5.3.3
```

Comment:
It would be lovely if these version numbers were (possibly environment?) variables all set in one place. This applies to several files, not just here.

Reply:
Agreed, that said this should only be a short-term workaround for a win32-specific bug in `jupyter_client`.